### ISSUE TITLE
Error loading IMDb info: Title not found.

### ISSUE THREAD

**Issue by Rouzax** [pymedusa/Medusa#4316]
I keep getting these warnings on a few of my shows

Medusa Info: | Branch: master                     Commit: 212cd1c8a350f2d5ca40f172ed5a227d9a5cb80f                     Version: 0.2.4                     Database: 44.9
-- | --
Python Version: | 2.7.14 (v2.7.14:84471935ed, Sep 16 2017, 20:25:58) [MSC v.1500 64 bit (AMD64)]
SSL Version: | OpenSSL 1.0.2k  26 Jan 2017
OS: | Windows-10-10.0.14393
Locale: | nl_NL.cp1252

```
2018-06-08 03:04:19 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 281621: Error loading IMDb info: Title not found. tt3566726 is a redirection imdb id
2018-06-08 03:04:12 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 328534: Error loading IMDb info: Title not found. tt6461726 is a redirection imdb id
2018-06-08 03:04:04 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 321239: Error loading IMDb info: Title not found. tt5834204 is a redirection imdb id
2018-06-08 03:03:55 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 320766: Error loading IMDb info: Title not found. tt5853176 is a redirection imdb id
2018-06-08 03:03:48 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 269653: Error loading IMDb info: Title not found. tt2712740 is a redirection imdb id
2018-06-08 03:03:42 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 80379: Error loading IMDb info: Title not found. tt0898266 is a redirection imdb id
2018-06-08 03:03:36 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 161511: Error loading IMDb info: Title not found. tt1586680 is a redirection imdb id
2018-06-08 03:03:30 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 320450: Error loading IMDb info: Title not found. tt5580540 is a redirection imdb id
2018-06-08 03:03:20 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 326288: Error loading IMDb info: Title not found. tt5618256 is a redirection imdb id
2018-06-08 03:03:14 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 279536: Error loading IMDb info: Title not found. tt4270492 is a redirection imdb id
2018-06-07 03:03:35 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 343253: Error loading IMDb info: Title not found. tt5232792 is a redirection imdb id
2018-06-07 03:03:26 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 296762: Error loading IMDb info: Title not found. tt0475784 is a redirection imdb id
2018-06-07 03:03:20 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 311969: Error loading IMDb info: Title not found. tt5511512 is a redirection imdb id
2018-06-07 03:03:01 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 315321: Error loading IMDb info: Title not found. tt4687880 is a redirection imdb id
2018-06-07 03:02:53 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 330134: Error loading IMDb info: Title not found. tt5531466 is a redirection imdb id
2018-06-07 03:02:46 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 269586: Error loading IMDb info: Title not found. tt2467372 is a redirection imdb id
2018-06-06 03:02:44 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 281593: Error loading IMDb info: Title not found. tt3339966 is a redirection imdb id
2018-06-06 03:02:32 WARNING  SHOWQUEUE-SEASON-UPDATE :: [212cd1c] 264586: Error loading IMDb info: Title not found. tt2372162 is a redirection imdb id
```

**Comment by sharkykh** [comment]
Already fixed in develop: #4253

**Comment by Rouzax** [comment]
Sorry, I did a search on open issues. Need to make a mental note to also search through the closed ones.
